test/util.js

```diff
@@ -166,9 +180,12 @@ async function prepareUnitTest(model, outputDirectory, parameters = {}) {
 
     if (parameters.transpilationCheck) {
         const tsFiles = paths.map(p => path.join(p, 'index.ts'))
+        // create a package.json w/ common dependencies in a higher dir so that they can be reused by many tests
+        createProject(path.resolve(outputDirectory, '../..'))
```

Comment:
without having checked, this reads a bit like it could cause rogue files in our monorepo setup. Did you perchance check that?

Reply:
From what I have seen, the tests are always executed in tmpdir, so to rouge source files.